Documentation/freebsd-instructions.md

```diff
@@ -0,0 +1,200 @@
+Build CoreCLR on FreeBSD
+======================
+
+This guide will walk you through building CoreCLR on FreeBSD and running Hello World.  We'll start by showing how to set up your environment from scratch.
+
+Environment
+===========
+
+These instructions are written assuming the 10.1-RELEASE, since that's the release the team uses.
+
+These instrutions assume you use the binary package tool `pkg` to install the environment. Compiling the dependencies from src using the ports tree might work too, but is untested.
```

Comment:
Hmm, I thought, "binary package tool" already made it clear? Yes pkg is the FreeBSD analog for apt-get, but there is also the ports tree. That is why people generally put an emphasis on "binary".